File: A-SOURCE_CODE/A-SERVICE/Administration/Models/Tables/Category.cs
using System.Collections.Generic;

namespace Administration.Models.Tables
{
    public class Category
    {
        #region Properties

        /// <summary>
        ///     Id of category.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        ///     Category name.
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        ///     When the category was created on service.
        /// </summary>
        public double Created { get; set; }

        /// <summary>
        ///     When the category was lastly modified.
        /// </summary>
        public double? LastModified { get; set; }

        #endregion

        #region Relationships

        /// <summary>
        ///     List of following relationships.
        /// </summary>
        public virtual ICollection<FollowCategory> BeingFollowed { get; set; }

        /// <summary>
        ///     List of posts which category contains.
        /// </summary>
        public virtual ICollection<Post> ContainPosts { get; set; }

        #endregion
    }
}

File: A-SOURCE_CODE/B-CLIENT/A-WEB/Administration/app/models/Category.ts
/*
* Category properties.
* */
import {Account} from "./Account";
export class Category{

    // Id of category.
    public id: number;

    // Category creator information.
    public creator: Account;

    // Name of category.
    public name: string;

    // When the category was created.
    public created: number;

    // When the category was lastly modified.
    public lastModified: number;
}